origami/client.py

```diff
@@ -242,6 +242,7 @@ async def get_kernel_session(
             self.file_session_cache[file_id] = session
             return session
 
+    @backoff.on_exception(backoff.expo, ReadTimeout, max_time=10)
```

Comment:
I think the initial request is already a 10s timeout. We should bump this to 60s max_time imo

Reply:
oh yeah, sure. I'll change the max_time in the other backoffs too.